### DISCUSSION TITLE
no precursor Lib.Q.Value > 1% ?

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#521] [Q&A]
I ran a primary DIA-NN search with MBR, took the generated library and overwrote the UniProt column with the Protein.Group column. Then I took the edited library and re-ran a DIA-NN search (larger set of raw files) with protein grouping disabled and. I plan to assemble proteins with the iQ package from the information present in the main report tsv. For that purpose, one should filter the precursors based on 'Lib.Q.Value' < 0.01 in that case, correct? (see #493 )
As far as I am aware, there is a run-specific q-value filter for precursors already in place for the main report tsv generation, but no filter for global precursor Q value or precursor library Q value. 

To my surprise, none of the precursor entries in the main report tsv had a 'Lib.Q.Value' below 1%. 
Q1: Is that expected ? Why is that? So why the filter?
Q2: If I take the DIA-NN MaxLFQ proteins form the main report, I need to filter them based on 'Lib.PG.Q.Value' < 1%, which does reduce some entries. By contrast, if I take the precursors (filtered for 'Lib.Q.Value' below 1% - but doesn't remove any) and use the iQ package, should I then filter the resulting iQ protein intensities based on the DIA-NN 'Lib.PG.Q.Value' < 1% ??

**Comment by vdemichev** [comment]
Yes, you can use the iQ package, but DIA-NN already reported protein quantities in PG.MaxLFQ and Genes.MaxLFQ.
Yes, Lib.Q.Value < 0.01.
Q1: Yes, every DIA library generated by DIA-NN itself will by default be filtered at 1% (precursor global q-value). This is important to ensure quality analysis.
Q2: Yes, in general Lib.PG.Q.Value filter should be applied. It's about IDs, so does not matter how intensities were calculated.

**Reply by JM-Bader** [reply]
corrigendum: I meant there are no precursor entries with a 'Lib.Q.Value' ABOVE 1% (so none will be removed by applying such filter)  -  but you understood it /answered it anyway. 
So in case of re-searches using a DIA-NN generated library, one has to filter proteins based on Lib.PG.Q.Value manually but nothing more (even if using precursors for iQ from main report etc.)

**Reply by vdemichev** [reply]
Yes
